docs/customApps.md

```diff
@@ -74,6 +74,30 @@ Before running analysis, make sure that you have `kantra` installed
 1. `./get_latest_kantra_cli.sh` # fetches 'kantra' our analyzer tool and stores it in ../bin
 1. `cd ..`
 
+
+To ensure the analysis uses the necessary custom rules, follow these steps:
+
+1. Custom Rules: Download or create your custom rules and place them in the samples/custom_rules directory.
+2. Ignore Default Rules: The analysis script is set to ignore default Kantra rules. You can review or adjust this in example/analyze.sh if needed.
```

Comment:
@Myrausman this information you have is not technically correct, as the section you are documenting will NOT ignore default rules.  That is why I gave you more context so you could fully understand the 2 places we run analysis.

The documentation for this section:
https://github.com/konveyor/kai/blob/1ed90050c210c5a3597b0ef1a073eb41f4843008/docs/customApps.md#run-analysis

Is talking about how to run the analysis for the sample data which Kai will consult. 
That analysis DOES NOT ignore default rules.

Please remove:
`2. Ignore Default Rules: The analysis script is set to ignore default Kantra rules. You can review or adjust this in example/analyze.sh if needed.`